src/models/db/users.js

```diff
@@ -1,27 +1,52 @@
-const db = require('./db')
+const bcrypt = require('bcrypt');
+const db = require('./db');
 
-const signUp = (user) => {
+const encryptPassword = (password) => {
+  const saltRounds = 10;
+  return bcrypt.hash(password, saltRounds);
+};
+
+const create = (user) => {
   return db.one(`
     INSERT INTO
-      users(first_name, email, encrypted_password)
+      users(full_name, email, password)
     VALUES
       ($1, $2, $3)
     RETURNING
       *
-  `, [user.first_name, user.email, user.password])
+  `, [user.full_name, user.email, user.password])
   .catch(console.log)
 };
 
-const signIn = (email, password) => {
+const signUp = (user) => {
+    return encryptPassword(user.password).then((hash) => {
+      user.password = hash
```

Comment:
be consistent about formatting promise chains. Do you line break for the `.then` or not?

Reply:
Complete 👍